README.md

```diff
@@ -34,14 +34,18 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+2. Name two big differences between ```display: block;``` and ```display: inline;```
+display block will put everything stacked, while display inline will put everything in the same line
+.
 
 3. What are the 4 areas of the box model?
+top bottom left right
 
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
+    perpendicular to the main axis
```

Comment:
Align item center aligns items along the cross axis up and down where y-axis (up and down) meets the x-axis (left and right).
            y-axis
             |
<----{item}---x-axis--->